GamePlaySystem/GamePlaySystem.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
GamePlaySystem I assume would be the Game in the future, is that is the case, is good to have them in the same repo? or Should it be in a separate repo?

Reply:
As you commented, when you use Unity, you don't upload the engine, you upload your project with scripts, then make sense to have 2 different repos. 
If we follow that kind of structure, Unity-like, we should create another repo with the scripts but with the Assets. And have the Engine repo and the Game repo. 
But this decision is better for the Lead. @mariofv